providers/css/webauthn-admin.css

```diff
@@ -0,0 +1,166 @@
+.webauth-register {
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex;
+	-webkit-box-align: center;
+	-ms-flex-align: center;
+	align-items: center;
+}
+.webauth-register .webauthn-error {
+	margin-left: 13px;
+}
+#webauthn-keys .busy,
+.webauth-register .busy {
+	color: #7f8284;
+	pointer-events: none;
+	-webkit-transition: opacity 0.3s ease;
+	-o-transition: opacity 0.3s ease;
+	transition: opacity 0.3s ease;
+	background-size: 30px 30px;
+	background-image: -o-linear-gradient(45deg, rgba(0, 0, 0, 0.1) 25%, transparent 25%, transparent 50%, rgba(0, 0, 0, 0.1) 50%, rgba(0, 0, 0, 0.1) 75%, transparent 75%, transparent);
+	background-image: linear-gradient(45deg, rgba(0, 0, 0, 0.1) 25%, transparent 25%, transparent 50%, rgba(0, 0, 0, 0.1) 50%, rgba(0, 0, 0, 0.1) 75%, transparent 75%, transparent);
+	-webkit-animation: barberpole 0.5s linear infinite;
+	animation: barberpole 0.5s linear infinite;
+}
+@-webkit-keyframes barberpole {
+	from {
+		background-position: 0 0;
+	}
+	to {
+		background-position: 60px 30px;
+	}
+}
+@keyframes barberpole {
+	from {
+		background-position: 0 0;
+	}
+	to {
+		background-position: 60px 30px;
+	}
+}
+.webauthn-key {
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex;
+	-ms-flex-wrap: wrap;
+	flex-wrap: wrap;
+	-webkit-box-align: last baseline;
+	-ms-flex-align: last baseline;
+	align-items: last baseline;
```

Comment:
We can probably center align this row.

```suggestion
	align-items: center;
```
| State | Before | After | 
| --- | --- |--- |
| | <img width="878" alt="Screenshot 2023-04-21 at 11 50 25 AM" src="https://user-images.githubusercontent.com/1657336/233529214-660c3b0d-cbf3-471e-b418-cb8f911b3b1b.png"> | <img width="867" alt="Screenshot 2023-04-21 at 11 50 16 AM" src="https://user-images.githubusercontent.com/1657336/233529216-ad6ee00c-a173-4d66-8341-57636ae3b0a5.png"> |
 | With Notice | <img width="873" alt="Screenshot 2023-04-21 at 11 50 35 AM" src="https://user-images.githubusercontent.com/1657336/233529210-6ac08388-b751-4e1f-987d-2154368ef3fb.png"> | <img width="874" alt="Screenshot 2023-04-21 at 11 50 51 AM" src="https://user-images.githubusercontent.com/1657336/233529205-8a68cba8-8731-4b66-9440-7ff237a2b758.png"> |